2-collectors/scala-stream-collector/examples/config.hocon.sample

```diff
@@ -166,6 +166,16 @@ collector {
       #host = "{{nsqHost}}"
       ## TCP port for nsqd, 4150 by default
       #port = {{nsqdPort}}
+
+      # Or EventHub
+      domainName = "servicebus.windows.net"
+
+      eventHubKeys {
+        goodAccessKeyName = ""
+        goodAccessKey = ""
+        badAccessKeyName = ""
+        badAccessKey = ""
+      }
```

Comment:
could you give more information regarding each and everyone of those fields in a comment?

Reply:
also, could you comment out the event hub config, the kinesis config is already in place, uncommented